migrations/v2.js

```diff
@@ -1,128 +1,120 @@
     return Boolean(components.length);
   });
   mutateContent('adapt-contrib-textInput - add _recordInteraction', async () => {
     components.forEach(component => component._recordInteraction = true);
     return true;
   });
   checkContent('adapt-contrib-textInput - check _recordInteraction added', async () => {
-    const isValid = components.every(component => hasKey(component, '_recordInteraction'));
+    const isValid = components.every(component => _.has(component, '_recordInteraction'));
     if (!isValid) throw new Error('_recordInteraction not added');
     return true;
   });
-  updatePlugin('adapt-contrib-textInput - update to v2.0.2', {name: 'adapt-contrib-textInput', version: '2.0.2', framework: '>=2.0.0'})
+  updatePlugin('adapt-contrib-textInput - update to v2.0.2', { name: 'adapt-contrib-textInput', version: '2.0.2', framework: '>=2.0.0' });
 });
 
 describe('adapt-contrib-textInput - v2.0.0 to v2.0.4', async () => {
-  whereFromPlugin('adapt-contrib-textInput - from v2.0.0 to v2.0.4', { name: 'adapt-contrib-textInput', version: '<2.0.4'});
+  whereFromPlugin('adapt-contrib-textInput - from v2.0.0 to v2.0.4', { name: 'adapt-contrib-textInput', version: '<2.0.4' });
   let components;
   whereContent('adapt-contrib-textInput - where missing _canShowModelAnswer', async content => {
-    components = getTextInputComponents(content).filter(component => !hasKey(component, '_canShowModelAnswer'));
+    components = getTextInputComponents(content).filter(component => !_.has(component, '_canShowModelAnswer'));
     return Boolean(components.length);
   });
   mutateContent('adapt-contrib-textInput - add _canShowModelAnswer', async () => {
     components.forEach(component => component._canShowModelAnswer = true);
     return true;
   });
   checkContent('adapt-contrib-textInput - check _canShowModelAnswer added', async () => {
-    const isValid = components.every(component => hasKey(component, '_canShowModelAnswer'));
+    const isValid = components.every(component => _.has(component, '_canShowModelAnswer'));
     if (!isValid) throw new Error('_canShowModelAnswer not added');
     return true;
   });
-  updatePlugin('adapt-contrib-textInput - update to v2.0.4', {name: 'adapt-contrib-textInput', version: '2.0.4', framework: '>=2.0.0'})
+  updatePlugin('adapt-contrib-textInput - update to v2.0.4', { name: 'adapt-contrib-textInput', version: '2.0.4', framework: '>=2.0.0' });
 });
 
 describe('adapt-contrib-textInput - v2.0.0 to v2.0.5', async () => {
-  whereFromPlugin('adapt-contrib-textInput - from v2.0.0 to v2.0.5', { name: 'adapt-contrib-textInput', version: '<2.0.5'});
+  whereFromPlugin('adapt-contrib-textInput - from v2.0.0 to v2.0.5', { name: 'adapt-contrib-textInput', version: '<2.0.5' });
   let components;
   whereContent('adapt-contrib-textInput - where has textInput components', async content => {
     components = getTextInputComponents(content);
```

Comment:
We now have the helpers `getComponents('textinput')`, `getCourse()` and `getConfig()`.

If you update your `adapt-migrations` installation to [`v1.1.0`](https://github.com/adaptlearning/adapt-migrations/releases/tag/v1.1.0) you'll be able to use them as direct imports.

You then don't need the custom helper `getTextInputComponents` nor the code `course = content.find(({ _type }) => _type === 'course');` throughout, you can use `getComponents('textinput')` and `getCourse()` respectively.

Import them as with the other functions from `adapt-migrations`:
```js
import {
  describe,
  whereContent,
  whereFromPlugin,
  mutateContent,
  checkContent,
  updatePlugin,
  getComponent,
  getCourse,
  getConfig
} from 'adapt-migrations';
```

Reply:
I saw these were coming but weren't ready at the time, will change accordingly now they are released.